metacat-main/src/main/java/com/netflix/metacat/main/manager/CatalogManager.java

```diff
@@ -38,7 +38,15 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 
 /**
- * Catalog manager.
+ * Catalog manager. This loads the catalogs defined as .properties files under the location defined by config property
+ * <code>metacat.plugin.config.location</code>.
+ * Usually there is a one-to-one mapping between a catalog and a data store. We could also have data stores
+ * (mostly sharded) addressed by a single catalog name.
+ * If a data store is sharded and needs to be represented by a single catalog name, then we will have multiple catalog
+ * property files, each referencing to its physical data store and having the same <code>catalog.name</code>.
+ * In this case, <code>catalog.name</code> and <code>metacat.schema.whitelist</code> will be used to point to the right
```

Comment:
schema. whitelist?  Why not using 'shards'?

Reply:
This was already defined. Concept of shards is higher than this. A shard will contain a list of databases.